### DISCUSSION TITLE
External Editor Support for Enhanced Flexibility

### DISCUSSION THREAD

**Discussion by lonicera** [isontheline/pro.webssh.net#1029] [Ideas]
Hello,

I have a suggestion regarding the application we are using. I've noticed that the built-in editor in the current version is quite functional, but adding support for an external editor could provide more flexibility for both me and other users.

Thank you!

**Comment by isontheline** [comment]
Hello @lonicera 👋

Thank you so much for your feedback 🙏
It seems related to : https://github.com/isontheline/pro.webssh.net/issues/763

Are you using WebSSH under iOS or macOS?
